iptables/iptables.go

```diff
@@ -15,8 +15,8 @@ func AddRule(appPort, metadataAddress, hostInterface, hostIP string) error {
 		return err
 	}
 
-	if hostIP == "" {
-		return errors.New("--host-ip must be set")
+	if hostIP == "0.0.0.0" {
+		return errors.New("iptables can't redirect to 0.0.0.0")
```

Comment:
Preventing 0.0.0.0

Reply:
shouldnt this bindIP?

Reply:
it's the name of the param of the func, renaming it for consistency

Reply:
✔️